src/cli/ImsBaseHandler.ts

```diff
@@ -55,14 +56,12 @@ export abstract class ImsBaseHandler implements ICommandHandler {
      *
      * @param {IHandlerParameters} commandParameters Command parameters sent to the handler.
      * @param {ImsSession} session The session object generated from the ims profile.
-     * @param {IProfile} imsProfile The ims profile that was loaded for the command.
      *
      * @returns {Promise<IIMSApiResponse>} The response from the underlying ims api call.
      */
     public abstract async processWithSession(
         commandParameters: IHandlerParameters,
-        session: ImsSession,
-        imsProfile: IProfile
+        session: ImsSession
     ): Promise<IIMSApiResponse>;
```

Comment:
I initially thought this could've been a breaking change but since we do not export anything on the `src/cli` folder, we are fine 😋 
Still good to do as part of vNext though since it's kind of a big change